backend/application/syson-application/src/test/java/org/eclipse/syson/application/controller/explorer/view/SysONExplorerTests.java

```diff
@@ -244,6 +249,66 @@ public void getRootContentWithHideMembershipsAndHideKerMLStandardLibraries() {
                 .verify(Duration.ofSeconds(10));
     }
 
+    @DisplayName("GIVEN an empty SysML Project, WHEN context menu is queried, THEN the menu is returned")
+    @Sql(scripts = { GeneralViewEmptyTestProjectData.SCRIPT_PATH }, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
```

Comment:
Please add `, config = @SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED))`
(it should be done for every single @Sql annotation)